queue.go

```diff
@@ -511,3 +510,17 @@ func (q *Queue) receiveJob(ctx context.Context) *amqp.Delivery {
 		return &job
 	}
 }
+
+func (q *Queue) setUrl() {
+	const urlString = "amqp://%s:%s@%s:%s/%s"
+	hostName := utils.GetEnvVar("RABBITMQ_HOSTNAME", "")
```

Comment:
please remove that `ugo-common/utils` dependency. In that case the standard `os.Getenv` works in the same way

Reply:
done